### PR TITLE
Add-badges-to-README.md

### PR DESCRIPTION
**Pull request by sarg3nt**

### CODE REVIEW

**Comment by sarg3nt** [comment]
@BrickCForce 
Ignore the build error with trivy. We are having issues with running out of ghcr pulls.  There's another PR that fixes it but for now don't let that stop you from approving.

File: README.md [hunk 1]
```diff
@@ -34,30 +52,79 @@ See the root `mise` config file at `home/vscode/.config/mise/config.toml` for al
 
 ## Using `mise` to Manage Go Versions
 
-1. Copy the `.mise.toml` file from the root of this repo to your projects repo root and modify it as needed.
-1. The provided `.devcontainer` will automatclly call `mise install` to install the custom versions of the applications.
-1. After the container is started and you shell into it you may need to call `mise use golang@<version>` to switch to the new version.  The included `help` explains how to do this in more detail and other `mise` commands you can use.
+1. Copy the `.mise.toml` file from the root of this repo to your project's root and modify it as needed.
+1. The provided `.devcontainer` will automatically call `mise install` to install the custom versions of the applications.
+1. After the container is started and you exec into it, `mise` will automatically install the app versions listed in the `.mise.toml` file.
+1. Other applications can also be installed by editing the `.mise.toml` file.
```
Comment (Stanrofl) [inline README.md:58]: Numbered steps were not incremented here.

File: README.md [hunk 2]
```diff
@@ -34,30 +52,79 @@ See the root `mise` config file at `home/vscode/.config/mise/config.toml` for al
...
+
+## Starship 
+
+Starship is a custom Power Line command prompt we include.
 
+![Starship Prompt](images/starship-prompt.png)
+
+For Starship to function properly you need a [Nerd Font](https://www.nerdfonts.com/) installed from [Nerd Fonts Downloads](https://www.nerdfonts.com/font-downloads).
+
+See [Initial Worksation Setup](#initial-worksation-setup) for instructions on downloading and installing the fonts.
```
Comment (Stanrofl) [inline README.md:120]: Typo for 'workstation' in both instances.

File: README.md [hunk 3]
```diff
@@ -1,21 +1,39 @@
...
+
+- [Included Tools](#included-tools)
+  - [Go](#go)
+  - [Utilities](#utilities)
+- [Using `mise` to Manage Go Versions](#using-mise-to-manage-go-versions)
+- [Included `.devcontainer` Config](#included-devcontainer-config)
+  - [Dev Container Setup](#dev-container-setup)
+  - [`dev.sh`](#devsh)
+- [Starship](#starship)
+- [Initial Worksation Setup](#initial-worksation-setup)
```
Comment (Stanrofl) [inline README.md:22]: Typo for 'workstation'? There seems to be 8 instances in total in this file.

**Review by Stanrofl — CHANGES_REQUESTED** [review]
Please fix spelling errors.

File: README.md (hunk 3 above)
Comment (sarg3nt) [reply README.md:22]: Fixed all

File: README.md (hunk 1 above)
Comment (sarg3nt) [reply README.md:58]: They are not supposed to be.  This is Markdown best practice.  The downstrem instance automatically numbers them correctly.  This makes adding new items in random places and moving items around the list easier.

File: README.md [hunk 4]
```diff
@@ -34,30 +52,79 @@ See the root `mise` config file at `home/vscode/.config/mise/config.toml` for al
...
+
+Do the following to use this example.
+
+1. Clone down the repository.  
+   `git clone https://github.com/sarg3nt/go-dev-container.git`
+2. Ensure your target project does not already have a `.devcontainer` directory.  If it does, you will either need to rename it for testing or delete it.
+3. Copy the `.devcontainer` directory to your project.
+4. Copy the following files to the root of your project.  
+   These are optional but encouraged.
+    - `.mise.toml`:  The config file for manageing specific versions of tooling you need for your Go project.
```
[the remark this answers is not in the document]
Comment (sarg3nt) [reply README.md:73]: Fixed

**Review by BrickCForce — APPROVED** [review]

**Review by Stanrofl — APPROVED** [review]

**Pull request merged**